lib/ansible/modules/cloud/amazon/s3_sync.py

```diff
@@ -105,6 +110,8 @@
     key_prefix: config_files/web
     file_change_strategy: force
     permission: public-read
+    region: us-east-1
```

Comment:
can you pull this out? the region shouldn't be required or implied- I know you had trouble with it, but that points to a misconfigured env var or boto config file.